Makefile

```diff
@@ -38,6 +38,8 @@ GITHUB_ACCESS_TOKEN := $(shell cat /builder/home/git-token 2> /dev/null)
 FEATURE_FLAG_TOKEN := $(shell cat /builder/home/feature-flag-token 2> /dev/null)
 CGO_ENABLED = 0
 
+REPORTS_DIR=reports/
```

Comment:
I'd go for `build/reports`, but having a directory to collect this all is a step forward.